<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage_Comment_Info</class>
 <widget class="QWidget" name="Manage_Comment_Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>570</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Manage_Comment_Info</string>
  </property>
  <widget class="QPushButton" name="btnModify">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>229</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>修改  </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.edit.add.png</normaloff>:/imgIcon/appbar.edit.add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color:black;
    background-color:rgb(238 , 238 , 238);
    border: 1px solid rgb(183 , 183 , 183);
	border-radius:5px;
}

QPushButton:hover
{
    color:black;
    background-color:rgb(228 , 240 , 250);
    border: 1px solid rgb(15 , 150 , 255);
}

QPushButton:pressed
{
    color:black;
    background-color:rgb(204 , 228 , 247);
    border: 1px solid rgb(1 , 84 , 153);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string>退出  </string>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.2.png</normaloff>:/imgIcon/appbar.close.2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QTableView" name="tableComment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>79</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtBrowserComment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>501</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>详细评语:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>14</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    color:white;
	background-color:rgb(50 , 204 , 194);
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(50, 184, 179);
}

QPushButton:pressed
{
    background-color:rgb(41, 144, 163);
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imageIcon.qrc">
     <normaloff>:/imgIcon/appbar.close.png</normaloff>:/imgIcon/appbar.close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>76</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>3</y>
     <width>76</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imgIcon/appbar.draw.pencil.reflection.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>174</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>评语管理</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imageIcon.qrc"/>
 </resources>
 <connections/>
</ui>
